calibration/CTDMOG/CGINS-CTDMOG-13561__20150621.csv

```diff
@@ -20,6 +20,6 @@ serial,name,value,notes
 37-13561,CC_ptcb1,0.0001,
 37-13561,CC_ptcb2,0,
 37-13561,CC_ptempa0,147.014,
-37-13561,CC_ptempa1,-0.06265311,
+37-13561,CC_ptempa1,-6.265312e-02,
 37-13561,CC_ptempa2,-6.518624e-07,
 37-13561,CC_wbotc,0,
```

Comment:
Looks good.

Reply:
Looks good (CD and JL) verified with QCT